rune_trie.go

```diff
@@ -22,7 +24,7 @@ func (trie *RuneTrie) Get(key string) interface{} {
 		node = node.children[r]
 		if node == nil {
 			// sfx additional condition to return prev value
-			if idx > 0 && key[idx-1] == '.' {
+			if idx > 0 && reflect.ValueOf(prevNode.value).String() == "*" {
```

Comment:
Why do we need reflect?
Can't we just use `string(prevNode.value) == "*"` ?

Reply:
you cannot do string(interface) to get string you would do prevNode.value.(string). Assuming that only string value is here. So to avoid crashes i have used reflect to make sure that it is string value and we compare with our wildcard "*" from Put ()